front/pages/poke/[wId]/index.tsx

```diff
@@ -314,6 +324,37 @@ const WorkspacePage = ({
     }
   );
 
+  const { submit: onToggleFeature, isSubmitting: isTogglingFeature } =
+    useSubmitFunction(async (feature: WhitelistableFeature, value: boolean) => {
+      try {
+        const r = await fetch(
+          `/api/poke/workspaces/${owner.sId}/features/${feature}`,
```

Comment:
Not a big fan of dealing with string here, mainly since we don't encode them, could we consider using the id or enforce the type of whitelistable feature to avoid some characters?

Reply:
Will make it a body param then 👍